src/repository/exercisesRepository.ts

```diff
@@ -325,6 +325,426 @@ export const db: Schema = {
+      Nós: 'dividimos',
+      'Eles/Elas/Vocēs': 'dividem',
+      exerciseLevel: 1
+    },
+    {
+      english: 'to take off',
+      infinitive: 'despir',
+      Eu: 'dispo',
+      Tu: 'despes',
+      'Ela/Ele/Você': 'despe',
+      Nós: 'despimos',
+      'Eles/Elas/Vocēs': 'despem',
+      exerciseLevel: 1
+    },
+    {
+      english: 'to correct',
+      infinitive: 'despir',
+      Eu: 'corrijo',
+      Tu: 'corriges',
+      'Ela/Ele/Você': 'corrige',
+      Nós: 'corrigimos',
+      'Eles/Elas/Vocēs': 'corrigem',
+      exerciseLevel: 1
+    },
+    {
+      english: 'to decide',
+      infinitive: 'decidir',
+      Eu: 'decido',
+      Tu: 'decides',
+      'Ela/Ele/Você': 'decide',
+      Nós: 'decidimos',
+      'Eles/Elas/Vocēs': 'decidem',
+      exerciseLevel: 1
+    },
+    {
+      english: 'to get',
+      infinitive: 'conseguir',
+      Eu: 'consigo',
+      Tu: 'consegues',
+      'Ela/Ele/Você': 'consegue',
+      Nós: 'conseguimos',
+      'Eles/Elas/Vocēs': 'conseguem',
+      exerciseLevel: 1
+    },
+    {
+      english: 'to serve',
+      infinitive: 'sirvo',
+      Eu: 'consigo',
+      Tu: 'serves',
+      'Ela/Ele/Você': 'serve',
+      Nós: 'servimos',
+      'Eles/Elas/Vocēs': 'servem',
+      exerciseLevel: 1
+    },
+    {
+      english: 'to feel',
+      infinitive: 'sentir',
+      Eu: 'sinto',
+      Tu: 'sentes',
+      'Ela/Ele/Você': 'sente',
+      Nós: 'sentimos',
+      'Eles/Elas/Vocēs': 'sentem',
+      exerciseLevel: 1
+    },
+    {
+      english: 'to translate',
+      infinitive: 'traduzir',
+      Eu: 'traduzo',
+      Tu: 'traduzes',
+      'Ela/Ele/Você': 'traduz',
+      Nós: 'traduzimos',
+      'Eles/Elas/Vocēs': 'traduzem',
+      exerciseLevel: 1
+    },
+    {
+      english: 'to translate',
+      infinitive: 'traduzir',
+      Eu: 'traduzo',
+      Tu: 'traduzes',
+      'Ela/Ele/Você': 'traduz',
+      Nós: 'traduzimos',
+      'Eles/Elas/Vocēs': 'traduzem',
+      exerciseLevel: 1
+    },
+    {
+      english: 'to leave',
+      infinitive: 'partir',
+      Eu: 'parto',
+      Tu: 'partes',
+      'Ela/Ele/Você': 'parte',
+      Nós: 'partimos',
+      'Eles/Elas/Vocēs': 'partem',
+      exerciseLevel: 1
+    },
+    {
+      english: 'to prefer',
+      infinitive: 'preferir',
+      Eu: 'prefiro',
+      Tu: 'preferes',
+      'Ela/Ele/Você': 'prefere',
+      Nós: 'preferimos',
+      'Eles/Elas/Vocēs': 'preferem',
+      exerciseLevel: 1
+    },
+    {
+      english: 'to drive',
+      infinitive: 'dirigir',
+      Eu: 'dirijo',
+      Tu: 'diriges',
+      'Ela/Ele/Você': 'dirige',
+      Nós: 'dirigimos',
+      'Eles/Elas/Vocēs': 'dirigem',
+      exerciseLevel: 1
+    },
+    {
+      english: 'to discuss',
+      infinitive: 'discutir',
+      Eu: 'discuto',
+      Tu: 'discutes',
+      'Ela/Ele/Você': 'discute',
+      Nós: 'discutimos',
+      'Eles/Elas/Vocēs': 'discutem',
+      exerciseLevel: 1
+    },
+    {
+      english: 'to conduct',
```

Comment:
exactly, conduzir is more like conduct, but hard to explain, maybe I would delete for this phase, take note of it and I can explain it to you later

Reply:
ok